tests/test_jobs.py

```diff
@@ -0,0 +1,53 @@
+from unittest.mock import Mock
+
+import pytest
+
+from pybuildkite.jobs import Jobs, LogFormat
+
+
+class TestBuildkite:
+    """
+    Test functionality of the Jobs class
+    """
+
+    @pytest.fixture
+    def fake_client(self):
+        """
+        Build a fake API client
+        """
+        return Mock(get=Mock())
+
+    def test_get_job_log_can_be_called(self, fake_client):
+        """
+        Test that we can request a jobs log when implicitly asking for text/html
```

Comment:
I assume we'll want to change these tests up to reflect the new interface

Reply:
I think I have had too little sleep to understand this today, could you clarify that?

Reply:
Should the tests now be using the enum instead of the actual strings is what i mean :)

Reply:
Oh i see what you’ve changed now. Awesome, ignore my comment